linux_os/guide/services/ssh/ssh_server/sshd_disable_root_password_login/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+title: 'Disable SSH root Login with a Password (Insecure)'
+
+description: |-
+To disable password-based root logins over SSH, add or correct the following
+line in <tt>/etc/ssh/sshd_config</tt>:
+<pre>PermitRootLogin prohibit-password</pre>
+
+warnings:
+    - general: |-
+         While this disables password-based root logins, direct root logins
+         through other means such as through SSH keys or GSSAPI will still be
+         permitted. Permitting any sort of root login remotely opens up the
+         root account to attack.
+         To fully disable direct root logins over SSH (which is considered a
+         best practice) and prevent remote attacks against the root account,
+         see CCE-XXXX: Disable root Logins
```

Comment:
@jcpunk please use the actual CCE for `Disable root Logins` instead of `CCE-XXXX`

Reply:
Can do